src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement.Core/Errors/ValidationErrors.cs

```diff
@@ -76,4 +76,130 @@ private static readonly ValidationErrorDescriptorFactory _factory
     /// </summary>
     public static ValidationErrorDescriptor AssignmentIsActiveInOneOrMoreDelegations { get; }
         = _factory.Create(10, $"Assignment is active in one or more delegations.");
+
+    /// <summary>
+    /// Gets a validation error descriptor
+    /// </summary>
+    public static ValidationErrorDescriptor NotAuthorizedForConsentRequest { get; }
+        = _factory.Create(19, "Not Authorized for ");
+
+    /// <summary>
+    /// Gets a validation error descriptor
+    /// </summary>
+    public static ValidationErrorDescriptor InvalidPersonIdentifier { get; }
+        = _factory.Create(20, "Invalid person identifier. No person with this identifier.");
+
+    /// <summary>
+    /// Gets a validation error descriptor
+    /// </summary>
+    public static ValidationErrorDescriptor InvalidOrganizationIdentifier { get; }
+        = _factory.Create(21, $"Invalid organization identifier. No organization with this identifier.");
+
+    /// <summary>
+    /// Gets a validation error descriptor
+    /// </summary>
+    public static ValidationErrorDescriptor InvalidValidToTime { get; }
+        = _factory.Create(22, $"The ValidTo time need to be in the future");
+
+    /// <summary>
+    /// Gets a validation error descriptor
+    /// </summary>
+    public static ValidationErrorDescriptor MissingConsentRight { get; }
```

Comment:
This should probably either be something like `ListEmpty`, or just use the `Required` from line 18.